### ISSUE TITLE
Please add more desciption

### ISSUE THREAD

**Issue by BieberleA** [HZDR-FWDF/ddafa#24]
- Papers (Kak et al, Hou et al ..... all papers, we looked for and used)
- Link to Tobias Conference Paper
- Link to Tobias CPC Paper (so far available :-))
- Add Interpolation Method, 
- Data processing strategy
- Memory management
- etc.

**Comment by j-stephan** [comment]
https://github.com/HZDR-FWDF/ddafa/wiki

What do you think of the proposed structure seen in the wiki?

I'm also not sure how Tobias' papers are relevant for ddafa, shouldn't they be added to ddrf instead?

**Comment by BieberleA** [comment]
Looks nice! In Tobias Paper the function of ddrf is presented. So, it makes perfectly sense to cite.

**Comment by j-stephan** [comment]
I agree that it should be cited, but wouldn't it make more sense to do this in ddrf's repository?

**Comment by BieberleA** [comment]
Yes, indeed.
